### DISCUSSION TITLE
How maximize box width

### DISCUSSION THREAD

**Discussion by cwik-tech** [JedWatson/react-select#5572] [Q&A]
I have a very simple question, how can I force the input box to **always** grow to max?
This is a sniped from my code:
`styles={{
            control: (provided, state) => ({
              ...provided,
              minWidth: '100%',
...rest of code....`
I tried width, maxWidth and minWdith:'100%' and still the same result:
for shorter imputs
![image](https://user-images.githubusercontent.com/54542292/222312620-071b8755-17a9-4906-8983-1708b520d733.png)
for longer inputs:
![image](https://user-images.githubusercontent.com/54542292/222312669-7a806b65-7613-421b-8b50-adc4902c91ac.png)
Any help would be appreciated!

**Comment by manjushsh** [comment]
Hi @cwikio , Please share code sandbox with above bug replicated. Thanks
